mtcr_ul/mtcr_ul_com.h

```diff
@@ -166,6 +166,8 @@ extern "C"
 
     int read_dword_from_conf_space(mfile* mf, u_int32_t offset, u_int32_t* data);
 
+    extern void mtcr_fix_endianness(u_int32_t* buf, int len);
```

Comment:
Please rationale the reason making function declaration an extern here.

Doesn't it extern by default?